.features/pending/feat-logging.md

```diff
@@ -0,0 +1,9 @@
+Component: General
```

Comment:
What is this file for?

Reply:
This is for #14640 - it provides us with a documented list of features ready for feature releases - so 3.8/4.0
will be easier.

It's under `pending` as it's pending release - if that's confusing it could be renamed. It would get moved into a different directory in the release process - see #14640's documentation.